examples/distributed/intranode/example_alltoall_route2x4.py

```
@@ -0,0 +1,348 @@
+    EAST = 3
+    SELF = 4
+
+
+@tilelang.jit(
+    pass_configs={
+        tilelang.PassConfigKey.TL_DISABLE_TMA_LOWER: True,
+        tilelang.PassConfigKey.TL_DISABLE_WARP_SPECIALIZED: True,
+    },
+    # debug_root_path="/home/zhengju.tang/tilescale/examples/distributed/debug/"
+)
+def torus_alltoall_xy(PE_num, X, Y, M, N, tile_M, tile_N, threads):
```

Ruff (0.14.14)

[warning] Unused function argument: `tile_N`

(ARG001)